extensions/ipynb/src/ipynbMain.ts

```diff
@@ -6,6 +6,25 @@
 import * as vscode from 'vscode';
 import { NotebookSerializer } from './notebookSerializer';
 
+// From {nbformat.INotebookMetadata} in @jupyterlab/coreutils
+type NotebookMetadata = {
+	kernelspec?: {
+		name: string;
+		display_name: string;
+		[propName: string]: unknown;
+	};
+	language_info?: {
+		name: string;
+		codemirror_mode?: string | {};
```

Comment:
Is this Python specific? Not clear to me whether this should be here or not either.

Reply:
This is part of the existing jupyter ipynb specification.